src/tests/unit/tests/popup/scripts/actions/popup-action-message-creator.test.ts

```diff
@@ -21,16 +21,17 @@ import { LaunchPanelType } from '../../../../../../popup/scripts/components/popu
 import { EventStubFactory } from '../../../../common/event-stub-factory';
 
 describe('PopupActionMessageCreatorTest', () => {
-    let eventStubFactory: EventStubFactory;
+    const eventStubFactory = new EventStubFactory();
+    const event = eventStubFactory.createKeypressEvent() as any;
```

Comment:
>event [](start = 10, length = 5)

Can this name be made a little more descriptive (maybe `stubKeypressEvent`?), now that it's being initialized further away from the use sites?

Reply:
done